.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  test:
+    name: ${{ matrix.os }} (R ${{ matrix.r_version }})
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
I really dislike that matrix format. Once you get past 3 or 4 jobs, it's hard to reason about all combinations of jobs that will exist. I think just explicitly including the sets of options is a lot clearer.

> Like maybe doing 3.6 so that it will automatically switch to future patches of 3.6, and either doing 4 or release to automatically keep up with new R versions.

Oh does the `setup-r` action support doing that? The examples here all use full `x.x.x` versions: https://github.com/r-lib/actions/tree/master/setup-r

Reply:
Yeah, doing `3.6` has worked for me for spongebob: https://github.com/jayqi/spongebob/blob/master/.github/workflows/tests-r-pkg.yml#L21-L24

Reply:
oh nice! Yeah I like that